docs/source/about/index.rst

```diff
@@ -0,0 +1,75 @@
+describe stochastic fate transitions, where each cell represents one state in the Markov chain. Markov chains describe
+memoryless transitions between system states through a probabilistic framework. Transition probabilities are summarized
+in a transition matrix :math:`T`, where :math:`T_{ij}`, describes the probability of transitioning from state :math:`i`
+to state :math:`j` in one step. In our context, each state corresponds to a cell. Markov chains are established tools in
+single-cell genomics and form the basis of many successful pseudotime approaches :cite:`haghverdi:16,setty:19`.
+
+By using Markov chains, we assume that cellular state transitions occur gradually and without memory. The former
+assumption implies that cells change their molecular state in small steps with many intermediate states which are
+captured in the data. This is a reasonable assumption for most biological systems. The latter assumption implies that a
+state transition depends only on the current molecular state and not on the history of states. This assumption is valid
+as CellRank describes average cellular dynamics, rather than any individual cell. Both assumptions form the basis of
+many of the previous successful TI approaches :cite:`haghverdi:16,setty:19,wolf:19`.
+
+.. _modularity:
+
+Modularity
+~~~~~~~~~~
+A typical CellRank workflow consists of two steps: **(i)** estimating cell-cell transition probabilities to set up a
+Markov transition matrix :math:`T`, and **(ii)** analyzing it using various tools to derive biological insights.
+Decoupling these two steps yields a powerful and flexible modeling framework as many analysis steps are independent
+of the construction of the transition matrix. For example, whether we use RNA velocity or a pseudotime to derive
+directed transition probabilities does not change how initial and terminal states are inferred or fate probabilities
+estimated. The general structure of the framework, corresponding to steps **(i)** and **(ii)**, is given by:
+
+* :ref:`Kernels <kernels>` that take multi-view single cell input data  and estimate a matrix of cell-cell transition
+  probabilities :math:`T`. Row :math:`i` in matrix :math:`T` contains the transition probabilities from cell :math:`i`
+  towards putative descendants. Therefore, all entries in the matrix are between 0 and 1, and rows sum to one.
+* :ref:`Estimators <estimators>` that take a cell-cell transition matrix :math:`T` computed using any kernel and apply
+  concepts from the theory of Markov chains to identify initial, terminal, and intermediate
+  :doc:`macrostates <../notebooks/tutorials/initial_terminal_states>` and compute
+  :doc:`fate probabilities <../notebooks/tutorials/fate_probabilities>`.
+
+Our main (and recommended!) estimator is based on *Generalized Perron Cluster Cluster Analysis* (GPCCA)
+:cite:`reuter:18,reuter:19`, a method originally developed to study conformational protein dynamics. CellRank uses a
```

Comment:
I would slightly change this:
"Our main (and recommended!) estimator is based on the *Generalized Perron Cluster Cluster Analysis* (GPCCA)
:cite:`reuter:18,reuter:19`, a method originally developed to study molecular dynamics."

Reply:
Ok, done.